draft-ietf-quic-http.md

```diff
@@ -1774,7 +1898,7 @@ removed. Because stream termination is handled by QUIC, an END_STREAM flag is
 not required.  This permits the removal of the Flags field from the generic
 frame layout.
 
-Frame payloads are largely drawn from {{!RFC7540}}. However, QUIC includes many
+Frame payloads are largely drawn from {{!HTTP2}}. However, QUIC includes many
```

Comment:
I'd prefer to handle that separately, since that's distinct from what I'm doing here and potentially sizable.  I've opened #2698 with a first pass at that change.